python/vyos/utils/process.py

```diff
@@ -60,6 +75,9 @@ def popen(command, flag='', shell=None, input=None, timeout=None, env=None,
     if not debug.enabled(flag):
         flag = 'command'
 
+    wrapper = get_wrapper(vrf, netns, auth)
+    command = wrapper + command
```

Comment:
```suggestion
    command = f'{wrapper} {command}'
```

I think it's better to use an approach that rules out any accidentally missing whitespace. If the whitespace is embedded  in the format string here, it's impossible to create a bad command by accidentally missing whitespace in different places.